src/audio/codec_adapter/CMakeLists.txt

```diff
@@ -13,4 +13,5 @@ endif()
 if(CONFIG_WAVES_CODEC)
 add_local_sources(sof codec/waves.c)
 sof_add_static_library(MaxxChrome ${CMAKE_CURRENT_LIST_DIR}/lib/release/libMaxxChrome.a)
+target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/)
 endif()
```

Comment:
Do we really need to add this include dir? I was expecting it is already added in the include path. Otherwise, other codecs will not compile.

Reply:
ah, good point - and ```include/sof``` is already in the $PATH

Reply:
@dbaluta, @lgirdwood 

root directory of Waves API `MaxxEffect` must be on the include path or it won`t compile
The reason is that it uses relative includes inside a, for example, in file `MaxxEffect_Revision.h`:
`
#include "MaxxEffect/MaxxEffect.h"
#include "MaxxEffect/MaxxStatus.h"
`

So there are two options here:
- put the MaxxEffect folder to a directory that is on the include path (this was an original CL but it was rejected)
- when building SOF with Waves codec add a folder with MaxxEffect to includes

Reply:
My suggestion is to put root directory 'MaxxEffect' in  `sof/src/include/sof/audio/codec_adapter/codec` and make sure
` sof/src/include/sof/audio/codec_adapter/codec` is in the include path. 

Does this work for you?

Reply:
yes, it will work, will do changes accordingly

Reply:
done